src/api/app/models/event_subscription/find_for_event.rb

```diff
@@ -54,7 +54,7 @@ def filter_and_convert_groups_without_emails_to_users(receivers)
       receivers.each do |receiver|
         case receiver
         when User
-          new_receivers << receiver
+          new_receivers << receiver if receiver.confirmed?
```

Comment:
@adrianschroeter we don't have a method `confirmed?`. Instead we have a `User#is_active?`

Reply:
ups, fixed